homeassistant/components/bosch_alarm/config_flow.py

```diff
@@ -98,11 +98,11 @@ async def async_step_user(
                 ConnectionRefusedError,
                 ssl.SSLError,
                 asyncio.exceptions.TimeoutError,
-            ):
-                _LOGGER.exception("Connection Error")
+            ) as e:
+                _LOGGER.error("Connection Error: %s", e)
                 errors["base"] = "cannot_connect"
-            except Exception:
-                _LOGGER.exception("Unexpected exception")
+            except Exception as e:  # noqa: BLE001
+                _LOGGER.error("Unexpected exception: %s", e)
```

Comment:
oh in this case we do want exception as in this case we truly don't know what happened